cypress/locators/multi-action-button/locators.js

```diff
@@ -1,3 +1,5 @@
 // component preview locators
-export const MULTI_ACTION_BUTTON_PREVIEW = 'div[data-component="multi-action-button"]';
-export const MULTI_ACTION_BUTTON_LIST = 'div[data-element="additional-buttons"]';
+export const MULTI_ACTION_BUTTON_COMPONENT = '[data-component="multi-action-button"]';
+export const MULTI_ACTION_BUTTON_LIST = '[data-element="additional-buttons"]';
+export const MULTI_ACTION_BUTTON_SUBTEXT = '[data-element="subtext"]';
```

Comment:
this locator is already defined in `./cypress/locators/button/locators.js`

Reply:
fixed